### ISSUE TITLE
Interactive mode for creating operator backed service

### ISSUE THREAD

**Issue by dharmit** [openshift/odo#2799]
/kind user-story

## User Story
As a user I want to interactively create an operator backed service so that I can provide the value to various parameters.

## Acceptance Criteria 
- [ ] `odo service create` command, which already takes you to interactive mode to start services based on Service Catalog, should also allow users to interactively create services from operators

## Links 
- Feature Request: #2613 

/kind user-story

**Comment by girishramnani** [comment]
Would you be able to complete this for v2 @dharmit ?

**Comment by dharmit** [comment]
No, I don't think this would be possible to do for v2. 2.1.0 might be a good target for this.

**Comment by kadel** [comment]
Creating operator-backed services was intentionally removed from odo v3. 

https://github.com/redhat-developer/odo/issues/5523 can be used as tracking issue I case we at some point decide to add it back 

/close
